packages/module/src/pipelines/utils/utils.ts

```diff
@@ -129,7 +129,7 @@ export const getEdgesFromNodes = (
   edgeType = DEFAULT_EDGE_TYPE,
   spacerEdgeType = DEFAULT_EDGE_TYPE,
   finallyNodeTypes: string[] = [DEFAULT_FINALLY_NODE_TYPE],
-  finallyEdgeType = DEFAULT_EDGE_TYPE
+  finallyEdgeType = DEFAULT_FINALLY_EDGE_TYPE
```

Comment:
we can't change the default setting. Applications won't necessarily have the new edge type in their component factory and this would break them.